setup.py

```diff
@@ -52,7 +52,7 @@
 DOWNLOAD_URL = "https://github.com/pydicom/pydicom/archive/master.zip"
 LICENSE = "MIT"
 VERSION = __version__
-# PACKAGE_DATA = {'pydicom': ['../tests/tests_files/*']}
+PACKAGE_DATA = {'pydicom': ['tests/test_files/*', 'tests/charset_files/*']}
 REQUIRES = []
```

Comment:
Ok, I see your point about the usage as example data - this makes sense. The larger package size also should not be a real problem nowadays, so I agree.